main.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <sstream>
+#include "Person.cpp"
+#include "Helper.cpp"
+#include "FileReader.cpp"
+#include "Phonebook.cpp"
+#include "Validator.cpp"
+using namespace std;
+
+Person *person(string name, string surname, string separator, string phoneNumber)
```

Comment:
Btw this method can be avoided, also it just creates with empty params this is wrong